qa/rpc-tests/sc_create_2.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/env python2
+# Copyright (c) 2014 The Bitcoin Core developers
+# Copyright (c) 2018 The Zencash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.authproxy import JSONRPCException
+from test_framework.util import assert_true, assert_equal, initialize_chain_clean, \
+    start_nodes, sync_blocks, sync_mempools, connect_nodes_bi, mark_logs, \
+    dump_sc_info, dump_sc_info_record
+import os
+import pprint
+from decimal import Decimal
+import json
+
+NUMB_OF_NODES = 2
+DEBUG_MODE = 0
+SC_COINS_MAT = 2
+
+
+class FakeDict(dict):
```

Comment:
Nice trick. You get a kind of iterable only dictionary, in the sense that you cannot access directly its keys, but in iterating it you can simulate duplicated keys